srunner/scenariomanager/scenarioatomics/atomic_criteria.py

```diff
@@ -1020,9 +1020,15 @@ def update(self):
         if location is None:
             return new_status
 
-        # Check if outside route lanes
-        self._is_outside_driving_lanes(location)
-        self._is_at_wrong_lane(location)
+        # Deactivate by other components
+        activate = py_trees.blackboard.Blackboard().get('AC_SwitchOutsideRouteLanesTest')
```

Comment:
Do something like the background manager, where you get the blackboard variable, adn then you set it to None

Reply:
Done